interface/patient_tracker/patient_tracker.php

```diff
@@ -206,9 +204,9 @@ function openNewTopWindow(newpid,newencounterid) {
 <span class="title"><?php echo xlt("Flow Board") ?></span>
 <body class="body_top" >
 <form method='post' name='theform' id='theform' action='<?php echo $action_page; ?>' onsubmit='return top.restoreSession()'>
-    <div id="flow_board_parameters">
```

Comment:
Because it prevents auto wrapping the parameters when the window is too narrow, I tried several thing and nothing worked but leaving it off.    I'm not CSS expert.   We never did get the CSS to work well in LibreEHR.